src/InteractiveWindow/Editor/InteractiveWindow_UIThread.cs

```diff
@@ -707,7 +704,7 @@ internal void AppendOutput(IEnumerable<string> output, int outputLength)
                 }
                 else if (insertLineBreak)
                 {
-                    InsertProjectionSpan(lineBreakProjectionSpanIndex, CreateTrackingSpan(_window._outputLineBreakBuffer, _window._lineBreakString));
+                    InsertProjectionSpan(lineBreakProjectionSpanIndex, _window._lineBreakString);
```

Comment:
No, it's a separate span with Inert content type. I've added a specific ReplSpanKind so that linebreaks are not confused with prompts though.